operator/src/main/java/oracle/kubernetes/operator/rest/model/ConversionRequest.java

```diff
@@ -34,11 +35,11 @@ public void setDesiredAPIVersion(String desiredAPIVersion) {
     this.desiredAPIVersion = desiredAPIVersion;
   }
 
-  public List<Object> getObjects() {
+  public List<Map<String,Object>> getObjects() {
```

Comment:
I think this method name is bit misleading since it's returning a list of map rather than list of objects.

Reply:
Isn't this returning the "objects" field from the conversion request? Each of those objects (i.e. Domains) is represented by a Map.

Reply:
This was confusing to me too because those maps can have entries that are not maps, but I believe that the top-level instance is always a Map. That would be consistent with the code we had previously where we always cast.

Reply:
ok, I was wondering if the method name can be renamed to something like `getDomains` or `getDomainMaps` to clarify it's intent ?

Reply:
Good point. As I understand it, the value was always a list of maps (the domains to convert, rendered as maps). I changed the type, but didn't change the name.

Reply:
And I see that the name "objects" is in the sample conversion request. So the next question is, how is the conversion request generated? Is "objects" specified by Kubernetes, or is it in the webhook definition?

Reply:
Kubernetes makes this request and defines the REST operation. The operator has to implement the operation that K8s has defined; we only configure the location of the endpoint.

Reply:
I believe we can leave the field name as `objects` but change the method names for getter and setter.